kolibri/plugins/facility_management/assets/src/views/UserPage/EditUserModal.vue

```diff
@@ -32,26 +32,43 @@
       v-model="newUsername"
     />
 
-    <KSelect
-      :label="$tr('userType')"
-      :options="userKinds"
-      v-model="newKind"
-    />
+    <template v-if="kind === UserKinds.SUPERUSER">
+      <h2 class="user-type header">
+        {{ $tr('userType') }}
+      </h2>
 
-    <fieldset class="coach-selector" v-if="coachIsSelected">
-      <KRadioButton
-        :label="$tr('classCoachLabel')"
-        :description="$tr('classCoachDescription')"
-        :value="true"
-        v-model="classCoachIsSelected"
+      <UserTypeDisplay
+        :userType="kind"
+        class="user-type"
       />
-      <KRadioButton
-        :label="$tr('facilityCoachLabel')"
-        :description="$tr('facilityCoachDescription')"
-        :value="false"
-        v-model="classCoachIsSelected"
+      <span class="super-admin-description">
+        Go to Device permissions to change this
```

Comment:
`$tr`

Reply:
Maybe link to this page too?

Reply:
> Maybe link to this page too?

On the map, see: https://www.figma.com/file/KxjIhoBljRKy71SARf1h7dO0/0.11-superuser-clarification?node-id=0%3A1